### PR TITLE
Storybook changes for Hide Attendee Names

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Conceptual docs for Hide Attendee Names

# Why
[TASK 3702169](https://skype.visualstudio.com/SPOOL/_workitems/edit/3702169)

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook/stories/HideAttendeeNames.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <img src="images/hideAttendees-PresenterView.png" alt="example of hide attendee names Presenter view" />
+  </Stack.Item>
+</Stack>
+
+**FAQs**
+
+- <b>How can I view the names of the attendees?</b>
+  <br />
+  Hide attendee names are enabled during Teams meetings to protect identities and privacy. Attendee names can only be
+  seen by organizers, co-organizers, and presenters. Organizer and presenter names and details are visible to everyone.
```
Comment (mgamis-msft) [inline packages/storybook/stories/HideAttendeeNames.stories.mdx:44]: nit: I would reword the last sentence to
"The names and details of organizers and presenters are visible to all participants."

File: packages/storybook/stories/HideAttendeeNames.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- <b>Can the hide attendee names option changed during the meeting?</b>
```
Comment (mgamis-msft) [inline packages/storybook/stories/HideAttendeeNames.stories.mdx:45]: nit: Can the hide attendee names option **be** changed during the meeting?

File: packages/storybook/stories/HideAttendeeNames.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  <br />
+  No, the hide attendee names option cannot be changed during the meeting. The option is set by the organizer before the
+  meeting starts.
+- <b>Why is pinning not working properly with the hide attendee names enabled?</b>
+  <br />
+  Pinning is not supported in case of promotion if i.e. attendee promoted to presenter. For other cases, pinning should
```
Comment (mgamis-msft) [inline packages/storybook/stories/HideAttendeeNames.stories.mdx:51]: nit: 
I would reword this to
"Pinning is not supported in the event of promotion, for example, when an attendee is promoted to presenter."

**Review by mgamis-msft — APPROVED** [review]

File: packages/storybook/stories/HideAttendeeNames.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+<Meta
+  id="hideAttendeeNames"
+  title="Concepts/Hide Attendee Names"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+## Hide Attendee Names
+
+Microsoft Teams meeting has introduced a Teams Premium meeting option called “Hide attendee names” which would mask the identity of attendees for other attendees in the meeting.
```
Comment (dmceachernmsft) [inline packages/storybook/stories/HideAttendeeNames.stories.mdx:13]: nit: `Microsoft Teams Meetings`

**Review by dmceachernmsft — APPROVED** [review]
